src/components/SearchBar.jsx

```diff
@@ -1 +1,31 @@
 // implement SearchBar component here
+import React from 'react';
+
+export default class SearchBar extends React.Component {
+  render() {
+    const { onSearchTextChange, onBookmarkedChange, onSelectedGenreChange } = this.props;
```

Comment:
Você pode adicionar o searchText, o bookmarkedOnly e o selectedGenre. 
```suggestion
    const { onSearchTextChange, onBookmarkedChange, onSelectedGenreChange, searchText, bookmarkedOnly, selectedGenre } = this.props;
```